app/models/search.rb

```diff
@@ -23,6 +23,7 @@ def #{method_name}=(value)
   def self.range_to_search(option)
     option.gsub!("&gt;", ">")
     option.gsub!("&lt;", "<")
+    option.downcase!
     match = option.match(/^([<>]*)\s*([\d -]+)\s*(year|week|month|day|hour)s?(\s*ago)?s*$/)
```

Comment:
Sorry I didn't get around to this sooner after being pinged over a month ago... >.>
There is indeed a magic Regex trick, just add the letter `i` after the last slash in the Regex expression to make the match case insensitive.

```ruby
match = option.match(/^([<>]*)\s*([\d -]+)\s*(year|week|month|day|hour)s?(\s*ago)?s*$/i)
```